sys-utils/lsns.c

```diff
@@ -1291,7 +1291,7 @@ static void __attribute__((__noreturn__)) usage(void)
 	fputs(_(" -r, --raw              use the raw output format\n"), out);
 	fputs(_(" -u, --notruncate       don't truncate text in columns\n"), out);
 	fputs(_(" -W, --nowrap           don't use multi-line representation\n"), out);
-	fputs(_(" -t, --type <name>      namespace type (mnt, net, ipc, user, pid, uts, cgroup, time)\n"), out);
+	fputs(_(" -t, --type[=<name>]    namespace type (mnt, net, ipc, user, pid, uts, cgroup, time)\n"), out);
 	fputs(_(" -T, --tree <rel>       use tree format (parent, owner, or process)\n"), out);
```

Comment:
The `--type` is defined as `required_argument`. I guess you want to modify `--tree` (like in .adoc).

Reply:
Sorry for taking the time to review my poor mistake. Updated.